tempesta_fw/http_parser.c

```diff
@@ -59,6 +63,31 @@ __field_finish(TfwHttpMsg *hm, TfwStr *field,
 }
 
 /**
+ * Check whether a character is CR or LF.
+ */
+#define IS_CR_OR_LF(c) (c == '\r' || c == '\n')
+
+/**
+ * Scans the initial @n bytes of the memory area pointed to by @s for the first
+ * occurance of EOL character.
+ */
+static inline unsigned char *
+memchreol(const unsigned char *s, size_t n)
+{
+	while (n) {
+		if (IS_CR_OR_LF(*s))
+			return (unsigned char *)s;
+		s++, n--;
+	}
+	return NULL;
+}
```

Comment:
You replaces vectorized ```memchr()``` by this singe-byte-step loop with 2 conditions and that's not good. Moreover, this is essentially bit specialized ```strcspn()```, which can be used instead. Please use [PCMPSTR](http://halobates.de/pcmpstr-js/pcmp.html).

There are also consider relating issue https://github.com/natsys/tempesta/issues/182, which probably partially fixed by the patch.